.gitignore

```diff
@@ -4,3 +4,4 @@ composer.lock
 .php_cs.cache
 .DS_Store
 Thumbs.db
+.idea/
```

Comment:
Imagine the mess if we add every IDE hidden files to the gitignore, maybe just add it to your global gitignore?